portfolio/src/main/webapp/script.js

```diff
@@ -374,4 +374,36 @@ function toggleCommentForm(loginStatus) {
 function commentsInit() {
   getComments();
   fetchLoginStatus();
+}
+
+/**
+ * Initializes the map.
+ */
+function mapInit() {
+  const homeIsolabellaCoords = { lat: 44.9067176, lng: 7.9096066 };
+  const universityCoords = { lat: 45.0623602, lng: 7.6599514 };
+
+  const map = new google.maps.Map(document.getElementById("map-container"), {
```

Comment:
recommend extracting the element retrieval (document.getElementById) into a variable